weave-js/src/components/PagePanel.tsx

```diff
@@ -130,7 +130,11 @@ const usePoorMansLocation = () => {
   return window.location;
 };
 
-const PagePanel: React.FC = props => {
+type PagePanelProps = {
+  browserType: string | undefined;
+};
+
+const PagePanel = ({browserType}: PagePanelProps) => {
   const weave = useWeaveContext();
   const location = usePoorMansLocation();
   const urlParams = new URLSearchParams(location.search);
```

Comment:
I know `exp` is super common (most of the other ones are dead/hacks). We might want to refactor so we have a `/show/` so it doesn't collide with the home.